### PR TITLE
Add logic for enabling the openshift test suite

### PR DESCRIPTION
**Pull request by umohnani8**
Add a build-rpms.sh script and Dockerfile that will
be used to enable the openshift aws test suites.

Will open the corresponding PR in openshift/release once this is in.

Signed-off-by: Urvashi Mohnani <umohnani@redhat.com>

### CODE REVIEW

**Comment by umohnani8** [comment]
@baude @lsm5 @mheon PTAL

**Comment by rhatdan** [comment]
LGTM

File: hack/lib/build/rpm.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+#  - None
+# Exports:
+#  - OS_RPM_VERSION
+#  - OS_RPM_RELEASE
+#  - OS_RPM_ARCHITECTURE
+function os::build::rpm::get_nvra_vars() {
+	# the package name can be overwritten but is normally 'origin'
+	OS_RPM_ARCHITECTURE="$(uname -i)"
+
+	# we can extract the pacakge version from the build version
```
Comment (TomSweeneyRedHat) [inline hack/lib/build/rpm.sh]: nit pacakge -> package

File: images/os/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+FROM fedora:29 AS build
```
Comment (TomSweeneyRedHat) [inline images/os/Dockerfile]: could we do latest or at least bump up to 31 or 32?

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests.

File: hack/lib/build/rpm.sh (hunk 1 above)
Comment (umohnani8) [reply hack/lib/build/rpm.sh]: Fixed

File: images/os/Dockerfile (hunk 2 above)
Comment (umohnani8) [reply images/os/Dockerfile]: All the openshift tests use f29 for injecting the rpm into the machine-os-content, so going to leave it at 29 for consistence.

**Comment by umohnani8** [comment]
@baude @mheon @rhatdan can we please get this in.

**Pull request merged**